RoboFile.php

```diff
@@ -10,7 +10,8 @@ class RoboFile extends \Robo\Tasks
     use DocumentationHelpers;
 
     const REPO_BLOB_URL = 'https://github.com/Codeception/Codeception/blob';
-    const STABLE_BRANCH = '4.1';
+    const BRANCH_4x = '4.1';
```

Comment:
Please use a generic name like `PREVIOUS_BRANCH`/`LEGACY_BRANCH` or make sure the constant is written in all uppercase `BRANCH_4_X`/ `BRANCH_4X`.